js/game.js

```diff
@@ -1,27 +1,54 @@
 $(document).ready(function() {
-		var points = 0;
-	$('.picture').on('click', function() {
+	
+	var points = 0;
+
+	var isCorrectAnswer = function() {
+		$('#popUp').find(".correct").removeClass("hide");
+		points++;
+	}	
+
+	var isWrongAnswer = function(){
+		$('#popUp').find(".incorrect").removeClass("hide");
+	}
 
+	$('#pictures').on('click', '[data-id="btp"]', isCorrectAnswer);
+	$('#pictures').on('click', '[data-id="briewood"]', isWrongAnswer);
+	$('#pictures').on('click', '[data-id="wando"]', isWrongAnswer); 
+ 	/*
```

Comment:
This might help in the future:
given:
```javascript
  var answers = {
    'btp': {
      isCorrect: true,
      imageSource: 'img/btp.jpg'
    },
    'briewood': {
      isCorrect: false,
      imageSource: 'img/briewood.jpg'
    },
   'wando': {
      isCorrect: false,
      imageSource: 'img/wando.jpg'
    },
```
```javascript
  for (place in answers) {
    $('#pictures').on('click', '[data-id="' + place + '"]', answers[place].isCorrect ? isCorrectAnswer : isWrongAnswer);
  }
```